.github/workflows/build-main.yml

```diff
@@ -35,7 +48,7 @@ jobs:
       uses: redhat-actions/push-to-registry@v2
       with:
         image: preflight
-        tags: ${{ steps.build-image.outputs.tags }}
```

Comment:
I'm not entirely sure how this will behave, when split into parallel sub-processes by `matrix`.
to be on the safe side and avoid any race conditions, I'm specifying the tag explicitly as built.